client/my-sites/sharing/buttons/preview-button.jsx

```diff
@@ -30,10 +31,23 @@ module.exports = React.createClass( {
 	},
 
 	getIcon: function() {
-		if ( 'string' === typeof this.props.button.genericon ) {
-			return <span className="noticon sharing-buttons-preview-button__glyph" aria-hidden="true">{ formatting.unicodeToString( this.props.button.genericon ) }</span>;
```

Comment:
I believe this was the only place we were using `unicodeToString`, so we may ought as well remove it from `formatting`?